### ISSUE TITLE
WAVE Errors

### ISSUE THREAD

**Issue by james-rae** [ramp4-pcar4/storylines-editor#567]
## Description

Saw the following non-highcharts WAVE errors. All are inside the slide-editor view.

1. The `Edit Project Metadata` button in the upper left is 💥 "Empty Button" 💥 
2. In the `Advanced` tab of the slide editor, the combo box in the blue header (defaults to "Text") 💥 fails colour contrasts 💥 
3. In the `Advanced` tab of the slide editor, the textarea that holds the raw config content is 💥 "Missing Form Label" 💥
